Package_SON.pretty/VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm.kicad_mod

```diff
@@ -0,0 +1,44 @@
+(module VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (layer F.Cu) (tedit 5B9F93A0)
+  (descr http://rohmfs.rohm.com/en/techdata_basic/ic/package/vson010v3030_1-e.pdf)
+  (tags "VSON-10 3x3mm")
+  (attr smd)
+  (fp_text reference REF** (at -0.006 2.572) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (at 0.248 -2.635) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -1.5 -1.5) (end -1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -1.5 1.5) (end 1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 1.5) (end 1.5 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 -1.5) (end -1.5 -1.5) (layer F.Fab) (width 0.1))
```

Comment:
It is recommended (in the KLC) to have a bevel of either 1mm or 25% of the shortest side of the part (whichever is less) on the fab layer to indicate pin 1, which is currently missing.